src/OpenFeature.Contrib.Providers.FeatureManagement/FeatureManagementProvider.cs

```diff
@@ -50,6 +50,19 @@ public override async Task<ResolutionDetails<bool>> ResolveBooleanValueAsync(str
         {
             var variant = await Evaluate(flagKey, context, CancellationToken.None);
 
+            if (variant == null)
+            {
+                var exists = false;
+                await foreach (var name in featureManager.GetFeatureNamesAsync().WithCancellation(cancellationToken))
+                {
+                    if (!flagKey.Equals(name, StringComparison.OrdinalIgnoreCase)) continue;
+                    exists = true;
+                    break;
+                }
+                var enabled = await featureManager.IsEnabledAsync(flagKey, context, cancellationToken);
+                return new ResolutionDetails<bool>(flagKey, exists ? enabled : defaultValue);
```

Comment:
This is the general guideline we are following at the moment for flags that don't exist.

```suggestion
                if (exists)
                {
                    return new ResolutionDetails<bool>(flagKey, enabled);
                } else {
                    return new ResolutionDetails<bool>(flagKey, defaultValue, ErrorType.FlagNotFound, Reason.Error);
                }
```

If you don't mind, please update the following lines that do evaluations in this file.

Reply:
Thanks for the suggestion, I've updated the code. Are these guidelines in the docs anywhere?

Reply:
Thanks, I am also looking to port this Feature Management Provider to Python so knowing those requirements are helpful